repositories/migrations/20240124111516_create_analytics_schema.sql

```diff
@@ -0,0 +1,37 @@
+-- +goose Up
+-- +goose StatementBegin
+-- create and make default the marble schema
+CREATE SCHEMA IF NOT EXISTS analytics;
+
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA analytics TO postgres;
+
+DO $$ BEGIN
+      CREATE USER analytics WITH PASSWORD 'default';
```

Comment:
Is it ok to let an hardcoded password like this ? The access to the db is handled in another way ?

Reply:
So the idea is that the default solution would be to create an analytics user before the migrations run, and expose an env variable for this.
I guess the best solution here is to create a user without password, which IIUC is possible (it just won't be possible to login as it)